schema/system/database_source.json

```diff
@@ -5,21 +5,26 @@
     "description": "information about a database source",
     "type": "object",
     "properties": {
-        "_id": {
+        "id": {
```

Comment:
Let's provide an example for clarity: - "For materialsproject.org an example ID is `mp-32`"